SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,119 @@
+
+Note that this SpatialOS documentation glossary assumes you are developing a SpatialOS game using the SpatialOS SDKs ([The Worker SDK and Platform SDK](#https://docs.improbable.io/reference/latest/shared/get-started/working-with-spatialos)),  so it may reference content relevant to that workflow only. While the SpatialOS SDK underpins the GDK for Unreal, the two workflows are not always the same.)
+
+### Access control list (ACL)
+In order to read from a [component](#spatialos-component), or make changes to a component, [workers](#workers) need to have [access](#authority), which they get through an access control list. 
+Access control lists are a component in the standard schema library: `EntityAcl`. Every [entity](#spatialos-entity) needs to have one. The ACL determines:
+* which types of workers have read access to an entity
+* for each component on the entity, which types of workers can have write access
+### Assembly
+An assembly is what’s created when you build your project. It contains all the files that your game uses.
+This includes executable files for the [client-workers](#workers) and [server-workers](#workers), and the assets these workers use (for example, textures used by a client to visualize the game).
+When you run a [cloud deployment](#deployment), you have to specify an assembly to use.
+
+### Authority
+Also known as “write access”.
+<br/>
+<br/>
+Write access:<br/>
+Many [workers](#workers) can connect to a [SpatialOS world](#spatialos-world). For each [component](#spatialos-component) on a [SpatialOS entity](#spatialos-entity), there can be no more than one worker with write access to it. This worker is the only one able to modify the component’s state and handle commands for it.
+Workers with write access are said to “have authority” and be “authoritative”; workers without write access are said to be “non-authoritative”.
+Which [types of workers](#worker-types) can have write access is governed by each entity’s [access control list (ACL)](#access-control-list-acl). This list specifies which worker actually has write access over an entity. The list is managed by SpatialOS, and can change regularly due to [load balancing (SpatialOS concept documentation)](https://docs.improbable.io/reference/latest/shared/glossary#load-balancing).
+<br/>
+<br/>
+Read access:<br/>
+Read access allows [workers](#workers) to know the state of a [SpatialOS component](#spatialos-component)  The [access control list (ACL)](#access-control-list-acl) also controls which workers have read-access to a [SpatialOS entity](#spatialos-entity). Read access does not allow a worker to change a component. Read access is at the entity level; if a worker can read from an entity, it is allowed to read from all components on that entity. 
```

Comment:
`Read access allows [workers](#workers) to know the state of a [SpatialOS component](#spatialos-component)  The [access control list (ACL)]` 
Missing full stop. :
`Read access allows [workers](#workers) to know the state of a [SpatialOS component](#spatialos-component). The [access control list (ACL)]`